frontend/packages/topology/src/elements/BaseGraph.ts

```diff
@@ -238,6 +248,19 @@ export default class BaseGraph<E extends GraphModel = GraphModel, D = any> exten
     if ('layout' in model) {
       this.setLayout(model.layout);
     }
+    const zoomExtents: [number, number] = [this.zoomExtents[0], this.zoomExtents[1]];
+    let updateExtents = false;
+    if ('minZoom' in model && typeof model.minZoom === 'number') {
+      zoomExtents[0] = +model.minZoom;
```

Comment:
Isn't `+variable` a cast to number? Wouldn't your if-statement typeof check and types make this check redundant?